src/Features/Core/Portable/SpellCheck/AbstractSpellCheckCodeFixProvider.cs

```diff
@@ -57,8 +57,8 @@ internal abstract class AbstractSpellCheckCodeFixProvider<TSimpleName> : CodeFix
             // Disable snippets from ever appearing in the completion items. It's
             // very unlikely the user would ever mispell a snippet, then use spell-
             // checking to fix it, then try to invoke the snippet.
-            var options = document.Options.WithChangedOption(
-                CompletionOptions.SnippetsBehavior, document.Project.Language, SnippetsRule.NeverInclude);
+            var options = (await document.GetOptionsAsync(cancellationToken).ConfigureAwait(false))
```

Comment:
in product code i really don't like awaits in subexpressions like this.  It's really hard to parse.  I'd prefer to just break into multiple statements.